Frontend/src/app/pages/home/home.page.scss

```diff
@@ -26,9 +26,10 @@
 }
 
 .card-container {
-  display: flex;
-  flex-wrap: wrap;
-  justify-content: space-around;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 200px);
```

Comment:
This automatically creates un undefined number of columns of 200px (because recipe cards images have a width of 200px), in function of the available width.